helpers/copy_vendor.sh

```diff
@@ -23,7 +23,7 @@ POST_RULES=$(tempfile)
 POST_SECTION_CREATED=0
 
 # Add read permission for some binaries under vendor mount
-source helpers/fixup-permissions.sh
+source droid-system-device/helpers/fixup-permissions.sh
```

Comment:
Thanks for finding this, but please perform a check if a local override exist, in order not to break the current XA2 adaptation.
And example could be found in https://github.com/mer-hybris/droid-hal-device/blob/a46bda94f5a11cb7d28821dda2e8f31b3dd07e22/droid-hal-device.inc#L349

The same applies below too.